services/ant-plus.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=MHP ANT+ mock wireless module
+
+[Install]
+WantedBy=default.target
+
+[Service]
+Restart=on-failure
+ExecStart=/bin/node ant_plus_logger
```

Comment:
which node version does `ant_plus_logger` require? I believe `/bin/node` would point to the node installed by root user which on the primary pi is version 10. pi user has node version 14. If we need 14, we can possibly use `sudo su - pi && /bin/node ant...`

Reply:
Looks like it runs under node 10 - If you're okay with this I might leave it as is. (your solution would still require the user to have manually installed node 14)
![image](https://user-images.githubusercontent.com/17876556/152135912-a8a79928-6e10-44e3-a3b0-8e7678e1343a.png)

Reply:
I'm fine if the script works okay with node 10.